test/suite/test_error_info02.py

```diff
@@ -57,21 +57,23 @@ def api_call_with_wt_rollback_wt_cache_overflow(self):
         # Open a session and cursor.
         cursor = self.session.open_cursor(self.uri)
 
-        # Start a transaction and insert a value large enough to trigger eviction app worker threads.
-        self.session.begin_transaction()
-        cursor.set_key("key_a")
-        cursor.set_value("a"*1024*5000)
-        cursor.update()
-        self.session.commit_transaction()
-
-        # Start a new transaction and attempt to insert a value. The very low cache_max_wait_ms
-        # value should cause the eviction thread to time out.
-        self.session.begin_transaction()
-        cursor.set_key("key_b")
-        cursor.set_value("b")
-
-        # This reason is the default reason for WT_ROLLBACK errors so we need to catch it.
-        self.assertRaisesException(wiredtiger.WiredTigerError, lambda: cursor.update())
+        # Start a transaction and insert a value large enough to trigger eviction app worker
+        # threads. Loop 1000 times to ensure that the eviction server is busy evicting, and the
+        # cache will be full when the application worker thread checks if eviction is needed.
+        for i in range(100):
+            self.session.begin_transaction()
+            cursor.set_key(str(i))
+            cursor.set_value(str(i)*1024*5000)
```

Comment:
@jiechenbo, should we make sure this does not generate overflow items?

Reply:
You are correct, weirdly it seems to be fine because we get the `wiredtiger.WT_ROLLBACK` back. I wonder if the overflow items only gets generated once we have reconciled already?

Reply:
Could be!

Reply:
I have confirmed this is true. It happens at reconciliation stage, it would not be relevant to it being eviction.